tests/phpunit/CRM/Group/Page/AjaxTest.php

```diff
@@ -650,4 +650,61 @@ public function hook_civicrm_aclGroup($type, $contactID, $tableName, &$allGroups
     }
   }
 
+  public function testEditAllGroupsACL() {
+    $this->setupEditAllGroupsACL();
+    $params = $this->_params;
+    $groups = CRM_Contact_BAO_Group::getGroupListSelector($params);
+    print_r($groups);
```

Comment:
@MegaphoneJon do we need the print_r here?

Reply:
@MegaphoneJon getting the integrated debugger going with PhpStorm is a pain to set up but so worth it!